rqd/rqd/rqmachine.py

```diff
@@ -279,11 +279,11 @@ def rssUpdate(self, frames):
                     child_statm_fields = self._getStatFields(
                         rqd.rqconstants.PATH_PROC_PID_STATM.format(pid))
                     pids[pid]['statm_size'] = \
-                        int(re.search(r"\d+", child_statm_fields[0]).group()) \
-                        if re.search(r"\d+", child_statm_fields[0]) else -1
+                        int(re.search(r"\d+", child_statm_fields[2]).group()) \
+                        if re.search(r"\d+", child_statm_fields[2]) else -1
                     pids[pid]['statm_rss'] = \
-                        int(re.search(r"\d+", child_statm_fields[1]).group()) \
-                        if re.search(r"\d+", child_statm_fields[1]) else -1
+                        int(re.search(r"\d+", child_statm_fields[3]).group()) \
+                        if re.search(r"\d+", child_statm_fields[3]) else -1
```

Comment:
I'll have a look, this was tested on RockyLinux9.1 and the fields seemed offset at that time.

Reply:
erratum, nothing to do with an offset, it was because _getStatFields was returning [None, None] + statFile before.
It was fixed in #1308 so I'll revert those changes.